aws/resource_aws_sns_platform_application_test.go

```diff
@@ -9,14 +9,68 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/sns"
 )
 
+func init() {
+	resource.AddTestSweepers("aws_sns_platform_application", &resource.Sweeper{
+		Name: "aws_sns_platform_application",
+		F:    testSweepSnsPlatformApplications,
+	})
+}
+
+func testSweepSnsPlatformApplications(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %w", err)
+	}
+	conn := client.(*AWSClient).snsconn
+	input := &sns.ListPlatformApplicationsInput{}
+	var sweeperErrs *multierror.Error
+
+	for {
+		output, err := conn.ListPlatformApplications(input)
```

Comment:
The SDK implements a paginated function 👍 [`ListPlatformApplicationsPages`](https://docs.aws.amazon.com/sdk-for-go/api/service/sns/#SNS.ListPlatformApplicationsPages)

Reply:
Thanks, must have missed that; I'll modify.

Reply:
Done.